src/components/ground-nav/ground-nav.stories.mdx

```diff
@@ -126,6 +126,115 @@ The menu’s contents may differ between projects. This example is based on the
         'https://github.com/cloudfour/pwastats/issues/new?title=Submission:+New+Story',
       buttonIcon: 'brands/github',
     }}
+    parameters={{
+      docs: {
+        source: {
+          code: `{% embed '@cloudfour/components/ground-nav/ground-nav.twig' with {
+  "menu": {
+    "items": [
+      {
+        "link": "/does",
+        "title": "What We Do",
+        "current": true
+      },
+      {
+        "link": "/believes",
+        "title": "Our Approach"
+      },
+      {
+        "link": "/made",
+        "title": "Our Work"
+      },
+      {
+        "link": "/thinks",
+        "title": "Articles"
+      },
+      {
+        "link": "/presents",
+        "title": "Speaking"
+      },
+      {
+        "link": "/contributes",
+        "title": "Labs"
+      },
+      {
+        "link": "/is",
+        "title": "Team"
+      },
+      {
+        "link": "/portland-device-lab",
+        "title": "Device Lab"
+      },
+      {
+        "link": "https://cloudfour-patterns.netlify.app/",
+        "title": "Patterns"
+      },
+      {
+        "link": "https://www.responsivefieldday.com/",
+        "title": "Responsive Field Day"
+      }
+    ]
+  },
+  "social": {
+    "items": [
+      {
+        "link": "https://twitter.com/cloudfour",
+        "title": "Follow us on Twitter",
+        "icon": "brands/twitter"
+      },
+      {
+        "link": "https://github.com/cloudfour",
+        "title": "Find us on GitHub",
+        "icon": "brands/github"
+      },
+      {
+        "link": "https://www.instagram.com/cloudfourpdx/",
+        "title": "Follow us on Instagram",
+        "icon": "brands/instagram"
+      },
+      {
+        "link": "https://www.linkedin.com/company/cloud-four",
+        "title": "Follow us on LinkedIn",
+        "icon": "brands/linkedin"
+      }
+    ]
+  },
+  "action": {
+    "lead_in": "Help us build this resource!",
+    "title": "Contribute a story",
+    "link": "https://github.com/cloudfour/pwastats/issues/new?title=Submission:+New+Story",
+    "icon": "brands/github"
+  },
+  "organization": {
+    "name": "Cloud Four, Inc.",
+    "address": {
+      "street_address": "510 SW 3rd Ave, Suite 420",
+      "address_locality": "Portland",
+      "address_region": "Oregon",
+      "postal_code": "97204",
+      "address_country": "USA"
+    },
+    "email": "info@cloudfour.com",
+    "telephone": "+1 (503) 290-1090",
+    "url": "https://cloudfour.com/",
+    "founding_date": "2007-12-13"
+  }
+} only %}
+  {% block kudos %}
+    <p class="c-ground-nav__kudos">
+      Hosted by <a href="https://www.netlify.com/">Netlify</a>, inspired by <a href="https://wpostats.com/">WPO Stats</a> and <a href="https://developers.google.com/web/showcase/">Google Web Showcase</a>
+    </p>
+  {% endblock %}
+  {% block legal %}
+    <p class="c-ground-nav__legal">
+      Designed by Cloud Four (<a href="https://github.com/cloudfour/pwastats/blob/master/LICENSE">License</a>)
+    </p>
+  {% endblock %}
+{% endembed %}
```

Comment:
Any reason why we didn't raw-load the `pwa-stats.twig` demo for this?

Reply:
I thought about that, but then the `with {}` arguments aren't included. If that's acceptable, then I can do the raw load instead. ¯\\\_(ツ)_/¯

Reply:
I think the trade-off is worth it to avoid having to maintain this source separately and so that the code example will be focused on the template markup.

Reply:
Done! :https://github.com/cloudfour/cloudfour.com-patterns/pull/1359/commits/b068fecd97753ad1d27d9604302f57b41abb7295